conda_build/skeletons/cran.py

```diff
@@ -146,7 +147,7 @@
 if {source_pf_bash}; then
   export DISABLE_AUTOBREW=1
   mv DESCRIPTION DESCRIPTION.old
-  grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
+  grep -va '^Priority: ' DESCRIPTION.old > DESCRIPTION
```

Comment:
This is a small but important change independent of the version support. Without the `-a` option, DESCRIPTION files with accented characters will not be processed correctly, and will very likely break the build.

Reply:
That's a good catch!